it/stmp_configuration/main.tf

```diff
@@ -0,0 +1,37 @@
+resource "ravel_configuration" "stmp_configuration" {
```

Comment:
Nit: I'm guessing this (and the filename) were supposed to be `smtp` instead of `stmp`, right?

Reply:
thanks, fixed

Reply:
Thanks for catching the file name.  Can we update the resource too?

Reply:
thanks, I didn't noticed that one :)